notebooks/algorithm-design/optimization.ipynb

```diff
@@ -0,0 +1,75 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Optimization loop\n",
```

Comment:
I think this topic lends itself to visual descriptions, there are some good examples in the [QML course](https://learn.qiskit.org/course/machine-learning/training-quantum-circuits).

I also think this should be plural
```suggestion
    "## Optimization loops\n",
```